pygmt/src/which.py

```diff
@@ -3,12 +3,11 @@
 """
 
 from pygmt.clib import Session
-from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+from pygmt.helpers import build_arg_list, fmt_docstring, is_nonstr_iter, use_alias
 
 
 @fmt_docstring
 @use_alias(G="download", V="verbose")
-@kwargs_to_strings(fname="sequence_space")
```

Comment:
Don't need to do it in this PR, but should we remove `sequence_space` from decorators.py at some point?

Reply:
Absolutely yes.